### ISSUE TITLE
Without Reps

### ISSUE THREAD

**Issue by BennyHz** [yancychy/DiffChIPL#4]
Hi! I wander if it's possible to run DiffChIPL without replication？

**Comment by yancychy** [comment]
No. DiffChIPL can not do it. But after you extract raw count data for each peak, you can use edgeR to compare single replicate. 
[https://www.bioconductor.org/packages/release/bioc/vignettes/edgeR/inst/doc/edgeRUsersGuide.pdf](url)

### 2.12 What to do if you have no replicates .
